src/cheatsheet.lua

```diff
@@ -103,8 +103,12 @@ function Cheatsheet._createShortcutBlocks(shortcutList)
             return string.reverse(a.hotkey) < string.reverse(b.hotkey)
         end)
 
+        local used_shortcuts = {}
         for _, shortcut in pairs(shortcuts) do
-            table.insert(block, { name = shortcut.name, hotkey = shortcut.hotkey })
+            if used_shortcuts[shortcut.key] == nil then
```

Comment:
It should be `hotkey` that should be used instead of `key` here (as well as on line 109):
```suggestion
            if used_shortcuts[shortcut.hotkey] == nil then
```

Tests should pass once this change is made 👍